qplock/rdma_spin_lock.cc

```diff
@@ -0,0 +1,77 @@
+#include "rdma_spin_lock.h"
+
+#include <infiniband/verbs.h>
+
+#include <atomic>
+#include <cstdint>
+#include <memory>
+#include <thread>
+
+#include "rome/rdma/connection_manager/connection_manager.h"
+#include "rome/rdma/memory_pool/remote_ptr.h"
+#include "util.h"
+
+namespace X {
+
+using ::rome::rdma::ConnectionManager;
+using ::rome::rdma::MemoryPool;
+using ::rome::rdma::remote_nullptr;
+using ::rome::rdma::remote_ptr;
+using ::rome::rdma::RemoteObjectProto;
+
+RdmaSpinLock::RdmaSpinLock(MemoryPool::Peer self,
+                         std::unique_ptr<MemoryPool::cm_type> cm)
+    : self_(self), pool_(self, std::move(cm)) {}
+
+absl::Status RdmaSpinLock::Init(MemoryPool::Peer host,
+                    const std::vector<MemoryPool::Peer> &peers) {
+    is_host_ = self_.id == host.id;
+    // auto capacity = 2 * sizeof(uint64_t);
```

Comment:
nit: try to avoid leaving commented lines if we don't need them (below as well). It's just clutter.